tools/SourceKit/tools/sourcekitd/lib/API/Requests.cpp

```diff
@@ -2430,9 +2433,24 @@ bool SKEditorConsumer::handleSourceText(StringRef Text) {
   return true;
 }
 
-bool SKEditorConsumer::handleSerializedSyntaxTree(StringRef Text) {
-  if (syntaxTreeEnabled())
-    Dict.set(KeySerializedSyntaxTree, Text);
+bool SKEditorConsumer::handleSyntaxTree(
+    swift::syntax::SourceFileSyntax SyntaxTree,
+    std::unordered_set<unsigned> ReusedNodeIds) {
+  if (Opts.SyntaxTransferMode == SyntaxTreeTransferMode::Off)
+    return true;
+
+  std::string SyntaxTreeString;
+  {
+    llvm::raw_string_ostream SyntaxTreeStream(SyntaxTreeString);
+    std::string SyntaxContent;
+    llvm::raw_string_ostream OS(SyntaxContent);
```

Comment:
These 2 lines (`OS` and its underlying `SyntaxContent`)are unused.

Reply:
Good catch. Forgot to remove them.